pyproject.toml

```diff
@@ -65,6 +65,9 @@ fallback_version = "unknown"
 # docstring-code-format = true
 # docstring-code-line-length = "dynamic"
 
+[tool.codespell]
+ignore-words-list = "aas,socio-economic,toi"
+
```

Comment:
I think there was talk about alphabetizing within `pyproject.toml`, which I'd be in support of so it's easier to find if a given tool is already there.

Reply:
*#557 originally contained https://github.com/tox-dev/pyproject-fmt to automate organizing that file but it was requested to be reverted back out.  `setuptools` is so central to python packaging that I have always seen it before other tools so that is why I placed `codespell` just after it.

Reply:
Thanks for the explanation! Totally makes sense on the order then.

> it was requested to be reverted back out.

So then is #557 needed?

Reply:
#557 is still providing a validator even though the formatting part of it was removed :rocket: